databaseQueries/userCourses.js

```
@@ -0,0 +1,131 @@
+          userCoursesDocuments = await database.models.userCourses.find(queryObject, projection).lean();
+        }
+
+        return resolve(userCoursesDocuments);
+      } catch (error) {
+        return resolve({
+          success: false,
+          message: error.message,
+          data: false,
+        });
+      }
+    });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)